elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -510,4 +514,8 @@ public Integer getQueryLimit() {
         // no limit
         return null;
     }
+
+    private static String getUniqueAlias(Class<?> entityType) {
```

Comment:
Should we rename this to something clearer? It's not exactly unique but merely distinct from `relationType`, right?